client/lib/browser-storage/index.ts

```diff
@@ -21,6 +21,10 @@ const STORE_NAME = 'calypso_store';
 
 const SANITY_TEST_KEY = 'browser-storage-sanity-test';
 
+function isAffectedSafari() {
+	return true;
```

Comment:
looking for a safe / small way to detect affected webkit versions, but having trouble. We may want to land this without the safari check.

The complication here is iOS in-app and Chromed browsers on iOS. All browsers on iOS are webkit under the hood, but they don't exhibit the same behavior when the quota is reacted. Chrome for iOS for example never prompts the user and just starts failing writes to idb.